### PR TITLE
removed the mattermost link and added google groups link in templates

### PR DESCRIPTION
**Pull request by girishramnani**
## What is the purpose of this change? What does it change?
see $SUBJECT

## Was the change discussed in an issue?
fixes https://github.com/openshift/odo/issues/1931

## How to test changes?
check the github templates

### CODE REVIEW

**Comment by amitkrout** [comment]
+1, Good stuff before GA
/approve

**Comment by dharmit** [comment]
That doesn't look like a publicly accessible Google group to me. @girishramnani @kadel

File: .github/ISSUE_TEMPLATE/Bug.md [hunk 1]
```diff
@@ -9,9 +9,9 @@ about: Report a problem with odo to help us resolve it and improve
 Welcome! - We kindly ask you to:
 
   1. Fill out the issue template below 
-  2. Use the chat and talk to us if you have a question rather than a bug or feature request.
+  2. Use the google group if you have a question rather than a bug or feature request.
 
-The chat room is at: https://chat.openshift.io/developers/channels/odo
+The group is at: https://groups.google.com/a/redhat.com/forum/#!forum/devtools-odo
```
Comment (mik-dass) [inline .github/ISSUE_TEMPLATE/Bug.md]: This google group is private to red hat employees

Comment (girishramnani) [reply .github/ISSUE_TEMPLATE/Bug.md]: I remember steve mentioning a group, I will find that out

Comment (kadel) [reply .github/ISSUE_TEMPLATE/Bug.md]: This is a different group.

Public odo group is [odo-users](https://groups.google.com/forum/#!forum/odo-users)

**Comment by kadel** [comment]
/hold

wrong google group.
The right one is https://groups.google.com/forum/#!forum/odo-users

**Pull request merged**